accounts/migrations/0001_initial.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 4.0.3 on 2022-05-26 17:37
```

Comment:
Are you sure we need this migrations file? I don't see a new model in this pr

Reply:
I've deleted it and "makemigrationed" when I had some db essues what do you suggest?

Reply:
I suggest remove the migrations file if the are not needed